.travis.yml

```diff
@@ -13,10 +13,20 @@ addons:
       - ant
       - ant-optional
 
+# set up a small VuFind and load some records
 before_script:
   - if [ ! -d downloads ]; then mkdir downloads; fi;
-  - if [ ! -f downloads/vufind-4.0.zip ]; then wget http://downloads.sourceforge.net/vufind/vufind-4.0.zip?use_mirror=osdn -O downloads/vufind-4.0.zip; fi;
-  - unzip downloads/vufind-4.0.zip
+  - if [ ! -f downloads/vufind-4.0.zip ]; then wget -nv http://downloads.sourceforge.net/vufind/vufind-4.0.zip?use_mirror=osdn -O downloads/vufind-4.0.zip; fi;
+  - unzip -q downloads/vufind-4.0.zip
+  - cd vufind-4.0
+  - ./solr.sh start
+  - VUFIND_HOME=$(pwd) ./import-marc-auth.sh tests/data/authority/authorityauth.mrc marc_auth_ils.properties
+  - VUFIND_HOME=$(pwd) ./import-marc-auth.sh ../tests/data/test-authorities.xml marc_auth_ils.properties
+  - VUFIND_HOME=$(pwd) ./import-marc.sh tests/data/authoritybibs.mrc
+  - VUFIND_HOME=$(pwd) ./index-alphabetic-browse.sh
+  - ./solr.sh stop
+  - cd ..
+  - mkdir -p build/deps # for some reason the setup ant task fails to create this directory
```

Comment:
Is it possible the ant task is trying to create `build/deps` all at once, but needs to create `build` first, then `build/deps`?

Reply:
The problem was using `pathconvert` before `build/deps` was created.